src/components/molecules/Comment/Comment.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+.comment {
+    margin: 1rem 1rem 2rem;
+
+    &__info {
+        display: flex;
+        gap: 2rem;
+        align-items: center;
+    }
+
+    &__text {
+        margin: 1rem 3rem;
+        padding: 1rem 1rem;
+        background-color: $comment-backgroundColor;
+        border-radius: 10px;
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    }
+}
+
+@include for-phone-only {
```

Comment:
On Main resposinve font change is handled, pull Main and you won't probably need these styles

Reply:
?? @Mariusz-web